deepchem/utils/save.py

```diff
@@ -116,6 +117,48 @@ def load_csv_files(filenames, shard_size=None, verbose=True):
         yield df
 
 
+def load_json_files(filenames: List[str],
+                    shard_size: Optional[int] = None,
+                    verbose: bool = True):
```

Comment:
Can you swap `verbose`/`log` out for using `logger.info`? I've been getting rid of the old-style verbose since it's much harder to use DeepChem in production settings with print statements.

Reply:
Done!